expression/expression.go

```diff
@@ -98,7 +98,7 @@ func getExpression(triggerExpression *TriggerExpression) (*govaluate.EvaluableEx
 		if triggerExpression.Expression == nil || *triggerExpression.Expression == "" {
 			return nil, fmt.Errorf("trigger_type set to expression, but no expression provided")
 		}
-		return getUserExpression(*triggerExpression.Expression)
+		return getUserExpression(triggerExpression)
```

Comment:
One point about check that I cannot stop thinking about is a performance. I want to see this functionality covered with benchmarks with various expressions tested(simple, complex, nested, nested with many layers) and comparison of performance before check implemented and after

Reply:
Yeah, I agree with you, these needs benchmarks log.

Reply:
I will add those after the logic consolidation